editor.html

```diff
@@ -86,7 +86,7 @@
             <h2 class="modal-title"><i class="fa fa-upload"></i> <strong>{{ load-title }}</strong></h2>
             <div class="load-drag-target" id="load-drag-target">
                 <input type="file" style="display: none" name="load-form-file-upload" id="file-upload-input">
-                <p>{{ instructions }}<br><a href="#" id="file-upload-link" class="load-drag-target load-toggle action">{{ toggle-file }}</a></p>
+                <p>{{ instructions }}<br><button aria-labelledby="file-upload-link" type="button" id="file-upload-link" class="load-drag-target load-toggle action">{{ toggle-file }}</button></p>
```

Comment:
Is `type="button"` required on a button?

Reply:
I think it's required in some cases to prevent default behaviour, see some discussion here https://stackoverflow.com/questions/41904199/whats-the-point-of-button-type-button

It looks to be of use to screenreaders and we use it for other buttons in this modal

Reply:
Okay, thanks for the clarification!